src/ring_theory/artinian.lean

```diff
@@ -428,4 +460,28 @@ begin
   rwa [← hn (n + 1) (nat.le_succ _)]
 end
 
+section localization
+
+variables {L : Type*} [comm_ring L] [algebra R L] (S : submonoid R) [is_localization S L]
+
+include S
+
+/-- Localizing an artinian ring can only reduce the amount of elements. -/
+theorem localization_surjective : function.surjective (algebra_map R L) :=
```

Comment:
It seems mathlib doesn't yet know a localization of an artinian ring is artinian, so we should probably prove this as a consequence (using [is_artinian_ring_of_surjective](https://leanprover-community.github.io/mathlib_docs/ring_theory/artinian.html#is_artinian_ring_of_surjective)).

Reply:
I changed the name of this to take dot notation and `fun_like`, btw. The `ring_equiv` version was made redundant as a result!

Reply:
The `localization` result can't be an instance due to metavariables too, btw

Reply:
`is_artinian_ring (localization M)` for a submonoid `M` could be registered as an instance though. Do we need it?